mne/bem.py

```diff
@@ -318,9 +318,7 @@ def make_bem_solution(surfs, verbose=None):
         logger.info('Homogeneous model surface loaded.')
     else:
         raise RuntimeError('Only 1- or 3-layer BEM computations supported')
-    if bem['surfs'][0]['np'] > 10000:
-        warn('The bem surface has %s data points. 5120 should be enough.'
-             % bem['surfs'][0]['np'])
+    _check_bem_size(bem)
```

Comment:
to save users some pain, you could also put this warning in `make_bem_model`, the lighter-weight step they likely use before making the solution